File: Python/bmimetric2.py
def bmi_metric(weight, height):
    BMI = round(weight/(height**2), 2)

    if BMI < 18.5:
        return BMI, "Underweight"
    elif BMI < 25.0:
        return BMI, "Normal"
    elif BMI < 30.0:
        return BMI, "Overweight"
    else:
        return BMI, "Obese"


if __name__ == "__main__":
    w = float(input("Please enter weight in kilograms: "))
    h = float(input("Please enter height in meters: "))
    bmi, status = bmi_metric(w, h)
    print(f"BMI is: {bmi}, Status is {status}")

File: C++/bmimetric2.cpp
#include<iostream>
#include <iomanip>
using namespace std;

int main() {
    double height, weight, bmi;

    std::cout << "Please enter weight in kilograms: ";
    std::cin >> weight;
    std::cout << "Please enter height in meters: ";
    std::cin >> height;

    bmi = weight/(height*height);
    if (bmi < 18.5){
        cout << setprecision(2) << std::fixed << "BMI is: " << bmi << ", Status is Underweight";
    } else if (bmi < 25.0) {
        cout << setprecision(2) << std::fixed << "BMI is: " << bmi << ", Status is Normal";
    } else if (bmi < 30.0) {
        cout << setprecision(2) << std::fixed << "BMI is: " << bmi << ", Status is Overweight";
    } else {
        cout << setprecision(2) << std::fixed << "BMI is: " << bmi << ", Status is Obese";
    }

    return 0;
}
